ding2.make

```diff
@@ -256,6 +256,9 @@ projects[opening_hours][patch][] = "http://www.drupal.org/files/issues/Issue620-
 ; Support series longer than a year.
 ; https://www.drupal.org/node/2194867
 projects[opening_hours][patch][] = "https://www.drupal.org/files/issues/opening_hours-2194867-D7.patch"
+; Add "hide if empty" option to field settings.
+; https://www.drupal.org/node/2820005
+projects[opening_hours][patch][] = "https://www.drupal.org/files/issues/opening-hours-2820005-hide-field-if-empty.patch"
```

Comment:
:+1: for submitting patches.

The opening hours maintainer mentioned that this patch will not be merged in the current form since it changes the default behavior.

As I see it we have the following options:

1. Accept the current patch and be aware that we have some work to do when we upgrade
2. Provide an updated patch which does not change default behavior and provide the configuration (strongarmed variable) needed to show the field even if empty.